web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -401,6 +441,32 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             return this._super.apply(this, arguments);
         },
 
+        /**
+         * Manages the keyboard events on the list. If the list is not editable, when the user navigates to
```

Comment:
this is an addition, should be treated as such: split the commit, add a new line in the changelog